communication.py

```diff
@@ -75,73 +75,83 @@ def pckParser(self, line):
+        height_pl = line[22] + line[23] + line[24] + line[25]
         [height_pl] = struct.unpack("f", height_pl)
-        height_car = line[25] + line[26] + line[27] + line[28]
+        height_car = line[26] + line[27] + line[28] + line[29]
         [height_car] = struct.unpack("f", height_car)
-        height_diff = line[29] + line[30] + line[31] + line[32]
+        height_diff = line[30] + line[31] + line[32] +  line[33]
         [height_diff] = struct.unpack("f", height_diff)
 
-        speed = line[33] + line[34] + line[35] + line[36]
+        speed = line[34] + line[35] + line[36] + line[37]
         [speed] = struct.unpack("f", speed)
-        tempe = line[37] + line[38] + line[39] + line[40]
+        tempe = line[38] + line[39] + line[40] + line[41]
         [tempe] = struct.unpack("f", tempe)
-        b_voltage = line[41] + line[42] + line[43] + line[44]
+        b_voltage = line[42] + line[43] + line[44] + line[45]
         [b_voltage] = struct.unpack("f", b_voltage)
 
-        latitude_pl = line[45] + line[46] + line[47] + line[48]
+        latitude_pl = line[46] + line[47] + line[48] + line[49]
         [latitude_pl] = struct.unpack("f", latitude_pl)
-        longitude_pl = line[49] + line[50] + line[51] + line[52]
+        longitude_pl = line[50] + line[51] + line[52] + line[53]
         [longitude_pl] = struct.unpack("f", longitude_pl)
-        altitude_pl = line[53] + line[54] + line[55] + line[56]
+        altitude_pl = line[54] + line[55] + line[56] + line[57] 
         [altitude_pl] = struct.unpack("f", altitude_pl)
 
-        altitude_car = line[57] + line[58] + line[59] + line[60]
-        [altitude_car] = struct.unpack("f", altitude_car)
-        latitude_car = line[61] + line[62] + line[63] + line[64]
-        [latitude_car] = struct.unpack("f", latitude_car)
-        longitude_car = line[65] + line[66] + line[67] + line[68]
-        [longitude_car] = struct.unpack("f", longitude_car)
-
-        status = line[69]
-        status = int.from_bytes(status, "little", signed=False)
-        yaw = line[70]+line[71]+line[72]+line[73]
+        yaw = line[58] + line[59] + line[60] + line[61]
         [yaw] = struct.unpack("f", yaw)
-        roll = line[74]+line[75]+line[76]+line[77]
+        roll = line[62] + line[63] + line[64] + line[65]
         [roll] = struct.unpack("f", roll)
-        pitch = line[78]+line[79]+line[80]+line[81]
+        pitch = line[66] + line[67] + line[68] + line[69]
         [pitch] = struct.unpack("f", pitch)
-        return_number = line[82]
-        return_number = int.from_bytes(return_number, "little", signed=False)
-        video_status = line[83]
-        video_status = int.from_bytes(video_status, "little", signed=False)
-        weather_forecast = line[84]
-        weather_forecast = int.from_bytes(weather_forecast, "little", signed=False)
+
+        takim_no = line[70] + line[71]
+        takim_no = int.from_bytes(takim_no, "little", signed=False)
+        takim_no = takim_no * 8
```

Comment:
this part is written for team number at last year. that number was more bigger than 2 bytes and after received that, we were multiplying it with 8. (so it was sended as divided by 8). this part is not necessary for now.